addons/osio-addon/src/main/java/io/fabric8/launcher/osio/identity/OsioIdentityRequests.java

```diff
@@ -0,0 +1,28 @@
+package io.fabric8.launcher.osio.identity;
+
+import java.util.Optional;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import io.fabric8.launcher.base.http.ExternalRequest;
+import io.fabric8.launcher.base.identity.IdentityHelper;
+import io.fabric8.launcher.base.identity.TokenIdentity;
+import io.fabric8.launcher.osio.EnvironmentVariables;
+import okhttp3.Request;
+
+public final class OsioIdentityRequests {
+
+    private OsioIdentityRequests() {
+        throw new IllegalAccessError("Helper class");
+    }
+
+    public static Optional<String> getServiceToken(final TokenIdentity osioToken, final String serviceName) {
```

Comment:
Why not make this method part of `OsioIdentityProvider`?

Reply:
IMO it's a Provider, so not it's responsability